File: lib/kotlin-mpp/src/commonMain/kotlin/com/sdercolin/utaformatix/data/Tempo.kt
package com.sdercolin.utaformatix.data

import kotlinx.serialization.Serializable

/**
 * Tempo label model
 * @param tickPosition Tick position of the tempo label
 * @param bpm Tempo in beats-per-minute
 */
@Serializable
data class Tempo(
    val tickPosition: Long,
    val bpm: Double
)

File: lib/csharp/UtaFormatix.Data/Tempo.cs
namespace UtaFormatix.Data;

/// <summary>Tempo label model.</summary>
///
/// <param name="TickPosition">Tick position of the tempo label.</param>
/// <param name="Bpm">Tempo in beats-per-minute.</param>
public record Tempo(long TickPosition, double Bpm);
